litestar/contrib/jwt/jwt_token.py

```diff
@@ -82,7 +89,12 @@ def decode(encoded_token: str, secret: str | dict[str, str], algorithm: str) ->
             payload = jwt.decode(token=encoded_token, key=secret, algorithms=[algorithm], options={"verify_aud": False})
             exp = datetime.fromtimestamp(payload.pop("exp"), tz=timezone.utc)
             iat = datetime.fromtimestamp(payload.pop("iat"), tz=timezone.utc)
-            return Token(exp=exp, iat=iat, **payload)
+            field_names = {f.name for f in dataclasses.fields(Token)}
+            extra_fields = set(payload.keys()).difference(field_names)
```

Comment:
```suggestion
            extra_fields = payload.keys() - field_names
```

that should work without creating a set explicitly.